.github/ISSUE_TEMPLATE/spike-template.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Spike Template
+about: Template for issues focused on research (spikes)
+title: 'SPIKE - '
+labels: Spike
+assignees: ''
+
+---
+
+## Description
```

Comment:
```suggestion
## Description

```
Extra linebreak after the headers.
https://www.markdownguide.org/basic-syntax/#heading-best-practices